x-pack/legacy/plugins/uptime/public/state/api/__tests__/ml_anomaly.test.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getMLJobId } from '../ml_anomaly';
+
+describe('ML Anomaly API', () => {
+  it('it generates a valid ML job ID', async () => {
```

Comment:
Can we break these out into different test cases? In an ideal world we'd do one `it` per test.

In some situations, where setup is complex and slow (like functional tests) breaking this rule makes sense.  For simple fast pure functions like this one it really cleans up the code to break the tests up more.

So, you'd have a single block like:

```javascript
it("should truncate long monitor IDs", () => {
  expect(getMLJobId(longAndWeirdMonitorId)).toHaveLength(64)
}
```

Reply:
i have updated the test.